troubadix/plugins/grammar.py

```diff
@@ -161,4 +161,12 @@ def check_for_false_positives(match: AnyStr, nasl_file: str) -> bool:
         # "with WITH stack unwinding"
         if "with WITH" in match:
             return True
+
+        # From 2008/debian/deb_1017_1.nasl
+        if (
+            "Harald Welte discovered that if a process issues a USB Request Block (URB)"
```

Comment:
That line needs to be wrapped see https://github.com/greenbone/troubadix/runs/6090207034?check_suite_focus=true

Reply:
Unfortunately i'm not familiar with Python internals on how such strings should be wrapped, can you give an example?

Reply:
for example
```python
            "Harald Welte discovered that if a process issues a "
            " USB Request Block (URB)"
```

Reply:
```suggestion
            "Harald Welte discovered that if a process issues a "
            "USB Request Block (URB)"
```